apps/code-examples/src/app/code-examples/forms/input-box/basic/input-box-demo.module.ts

```diff
@@ -1,22 +1,30 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
+import { SkyIdModule } from '@skyux/core';
 import { SkyDatepickerModule } from '@skyux/datetime';
 import { SkyInputBoxModule } from '@skyux/forms';
 import { SkyHelpInlineModule } from '@skyux/indicators';
-import { SkyThemeModule, SkyThemeService } from '@skyux/theme';
+import { SkyBoxModule, SkyFluidGridModule } from '@skyux/layout';
+import { SkyPopoverModule } from '@skyux/popovers';
+import { SkyEmailValidationModule } from '@skyux/validation';
 
 import { InputBoxDemoComponent } from './input-box-demo.component';
 
 @NgModule({
   imports: [
     CommonModule,
+    ReactiveFormsModule,
+    SkyBoxModule,
     SkyDatepickerModule,
-    SkyInputBoxModule,
-    SkyThemeModule,
+    SkyEmailValidationModule,
+    SkyFluidGridModule,
     SkyHelpInlineModule,
+    SkyIdModule,
+    SkyInputBoxModule,
+    SkyPopoverModule,
   ],
   declarations: [InputBoxDemoComponent],
   exports: [InputBoxDemoComponent],
-  providers: [SkyThemeService],
 })
 export class SkyInputBoxDemoModule {}
```

Comment:
I know you didn't do this - but we tend to not prefix things in our demos with `Sky`. Would you mind tweaking that so that everything matches in naming convention?